src/utils/string/index.ts

```diff
@@ -24,11 +24,48 @@ export const msToDuration = (ms: number): string => {
   return `${milliseconds}ms`;
 };
 
+export const stringifyNanoseconds = (input, skipDayMax, skipSecMax) => {
```

Comment:
needs unit test

Reply:
This is copied from [here](https://github.com/evergreen-ci/evergreen/blob/master/public/static/app/waterfall/tasks_grid.jsx#L93). I just changed var to const and things like that.